test/node/replumb/require_test.cljs

```diff
@@ -116,6 +131,7 @@
         (is (valid-eval-result? out) "(require 'alterable.core) and alterable.core/b should be a valid result")
         (is (= 'cljs.user (repl/current-ns)) "(require 'alterable.core) and alterable.core/b should not change namespace")
         (is (= "\"pre\"" out) "(require 'alterable.core) and alterable.core/b should return \"pre\""))
+
```

Comment:
These lines are added on purpose so that we can navigate easily (it is a simple `CTRL-up/down on emacs)